.eslintrc.cjs

```diff
@@ -32,6 +32,14 @@ module.exports = {
 
     "@typescript-eslint/no-non-null-assertion": "warn",
   },
+  overrides: [
+    {
+      files: ["examples/**/*"],
+      rules: {
+        "unused-imports/no-unused-vars": "off",
```

Comment:
I am guessing that in the `examples` directory you might want to keep unused vars? I'm happy to just fix the actual issues instead of disabling the rule if that's preferred 🙂

Reply:
Yeah i guess that's fine. 
Everything in `examples` is there for e2e and as example, it make sense that we'll have unused variable sometimes.